src/main/java/by/onliner/util/MoveTo.java

```diff
@@ -0,0 +1,21 @@
+package by.onliner.util;
+
+import by.onliner.driver.DriverManager;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+/***
+ * Wrapper class for actions moves by mouse to the defined element
+ */
+public class MoveTo {
+
+    protected static Actions actions = new Actions(DriverManager.getDriver());
+
+    public static void moveToElementAndClick(WebElement webElement) {
+        actions.moveToElement(webElement).click(webElement).build().perform();
+    }
+
+    public static void clickElement(WebElement webElement) {
```

Comment:
Are you sure this method is needed? And are you sure it's supposed to be in this class?

Reply:
OK, this method was deleted